### PR TITLE
Import platform

### PR DESCRIPTION
**Pull request by ldgeo**

### CODE REVIEW

File: cli_li3ds/import_platform.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            'name': 'cameraMetaData.json',
+            'type': 'group'})
+
+        ref_camera_group = api.Referential(
+            camera_group,
+            {'name': 'base'})
+
+        ref_lidar = api.Referential(
+            lidar,
+            {'name': 'ref lidar'})
```
Comment (elemoine) [inline cli_li3ds/import_platform.py]: We could name it "ref lidar cartesian" instead.

File: cli_li3ds/import_platform.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            func_signature=['mat4x3']
+        )
+
+        transfo_ins2camera_group = api.Transfo(
+            ref_ins,
+            ref_camera_group,
+            name='identity',
+            transfo_type=identity,
+            parameters=[
+                {'mat4x3': [1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0]}
```
Comment (elemoine) [inline cli_li3ds/import_platform.py]: I think identity is `[1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0]`.

File: cli_li3ds/import_platform.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            ])
+        transfo_lidar2ins = api.Transfo(
+            ref_lidar,
+            ref_ins,
+            name='identity',
+            transfo_type=identity,
+            parameters=[
+                {'mat4x3': [1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0]}
```
Comment (elemoine) [inline cli_li3ds/import_platform.py]: Ditto on the identity matrix.

File: cli_li3ds/import_platform.py (hunk 2 above)
Comment (ldgeo) [inline cli_li3ds/import_platform.py]: Done

File: cli_li3ds/import_platform.py (hunk 1 above)
Comment (ldgeo) [inline cli_li3ds/import_platform.py]: Done

File: cli_li3ds/import_platform.py (hunk 3 above)
Comment (ldgeo) [inline cli_li3ds/import_platform.py]: Done

File: cli_li3ds/import_platform.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            'name': 'cameraMetaData.json',
+            'type': 'group'})
+
+        ref_camera_group = api.Referential(
+            camera_group,
+            {'name': 'base'})
+
+        ref_lidar = api.Referential(
+            lidar,
+            {'name': 'ref lidar cartesian'})
```
Comment (elemoine) [inline cli_li3ds/import_platform.py]: Do you mind removing "ref" here in the referential name? It's pattern we haven't used elsewhere.

File: cli_li3ds/import_platform.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            camera_group,
+            {'name': 'base'})
+
+        ref_lidar = api.Referential(
+            lidar,
+            {'name': 'lidar cartesian'})
+
+        ref_ins = api.Referential(
+            ins,
+            {'name': 'ref ins'})
```
Comment (elemoine) [inline cli_li3ds/import_platform.py]: s/ref ins/ins

**Comment by elemoine** [comment]
New error at deploy time (on the li3ds server):

```
Error: "specifications" mismatch in sensor "lidar" ("{}" vs "None")
```

I will have a look tomorrow.

**Comment by elemoine** [comment]
The issue is related to an incompatibility between `import-platform` and `import-ept`. `import-ept` creates a lidar sensor with `specifications` set to `{}` while `import-platform` creates a lidar sensor with `specifications` set to `NULL`.

**Pull request merged**